MovieGallery/Pages/Search.cshtml

```diff
@@ -1,10 +1,10 @@
 ﻿@page
 @model MovieGallery.Pages.SearchModel
 @{
-    ViewData["Title"] = "Search by year";
+    ViewData["Title"] = "Search";
 }
 
-<h2> Search your favourite movies and shows</h2>
+<h2> Search your favourite movies and shows by year</h2>
```

Comment:
When we click on the search page, I have added in the description to search by year for better readability for the users. This will be the first thing which users will read when redirected to this page.